src/ziggurat/kafka_consumer/consumer.clj

```diff
@@ -11,6 +11,7 @@
    :max-poll-records 500
    :session-timeout-ms-config 60000
    :enable-auto-commit true
+   :default-api-timeout-ms 60000 ;; default as per kafka documentation
```

Comment:
Aren't all of them default as per kafka documentation?

Reply:
yep all of them are default. removing comment....